app/ui/src/main/java/com/fsck/k9/ui/settings/AboutActivity.kt

```diff
@@ -1,42 +1,75 @@
 package com.fsck.k9.ui.settings
 
+import java.util.Calendar
+
 import android.content.Context
 import android.content.Intent
 import android.content.pm.PackageManager
+import android.net.Uri
 import android.os.Bundle
+import android.support.v4.app.Fragment
+import android.support.v7.widget.LinearLayoutManager
+import android.support.v7.widget.RecyclerView
+import android.view.LayoutInflater
 import android.view.Menu
 import android.view.MenuItem
+import android.view.View
+import android.view.ViewGroup
 import android.webkit.WebView
+import android.widget.TextView
 import com.fsck.k9.activity.K9Activity
 import com.fsck.k9.ui.R
 import de.cketti.library.changelog.ChangeLog
+import kotlinx.android.synthetic.main.about.*
+import kotlinx.android.synthetic.main.about_library.view.*
+
 import timber.log.Timber
-import java.util.Calendar
 
-class AboutActivity : K9Activity() {
-    private lateinit var webView: WebView
+private data class Library(val name: String, val URL: String, val license: String = "")
```

Comment:
Hmm... as far as I can see all libraries have licenses so I think we can drop the default (` = ""`).

Reply:
Done in 7b15649